src/Aspire.Dashboard/Otlp/Http/OtlpHttpEndpointsBuilder.cs

```diff
@@ -0,0 +1,200 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Buffers;
+using System.IO.Pipelines;
+using System.Net.Http.Headers;
+using Aspire.Dashboard.Authentication;
+using Google.Protobuf;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using OpenTelemetry.Proto.Collector.Logs.V1;
+using OpenTelemetry.Proto.Collector.Metrics.V1;
+using OpenTelemetry.Proto.Collector.Trace.V1;
+
+namespace Aspire.Dashboard.Otlp.Http;
+
+public static class OtlpHttpEndpointsBuilder
+{
+    public const string ProtobufContentType = "application/x-protobuf";
+    public const string JsonContentType = "application/json";
+
+    public static void ConfigureHttpOtlp(this IEndpointRouteBuilder endpoints)
```

Comment:
nit: 
```suggestion
    public static void MapOtlpEndpoints(this IEndpointRouteBuilder endpoints)
```

Map* is the verb we use for adding routes. Configure is for options.

Reply:
I think I came up with the `Map*` verb back in .NET Core 3 😋